pkg/operator/etc/config.go

```diff
@@ -19,6 +19,9 @@ type Config struct {
 	HealthProbeBindAddress        string        `env:"OPERATOR_HEALTH_PROBE_BIND_ADDRESS" envDefault:":9090"`
 	LogDevMode                    bool          `env:"OPERATOR_LOG_DEV_MODE" envDefault:"false"`
 	CISKubernetesBenchmarkEnabled bool          `env:"OPERATOR_CIS_KUBERNETES_BENCHMARK_ENABLED" envDefault:"true"`
+	LeaderElectionEnabled         bool          `env:"OPERATOR_LEADER_ELECTION_ENABLED" envDefault:"false"`
+	LeaderElectionNamespace       string        `env:"OPERATOR_LEADER_ELECTION_NAMESPACE"`
```

Comment:
Can we use existing OPERATOR_NAMESPACE instead? Is there any scenario where we may want to save lock in the different namespace than the operator itself?

Reply:
We should also document any new envs in `docs/operator/configuration.md`

Reply:
I think we can use the `OPERATOR_NAMESPACE`.